src/com/projectkorra/projectkorra/airbending/Tornado.java

```diff
@@ -152,6 +156,14 @@ private void rotateTornado() {
 							if (Commands.invincible.contains(((Player) entity).getName())) {
 								continue;
 							}
+							
+							Player trg = (Player) entity;
+							BendingPlayer bPlayer = BendingPlayer.getBendingPlayer(trg);
+							if (player.canBend(CoreAbility.getAbility(AirSpout.class))) {

```

Comment:
This line wouldn't have compiled properly if you did this in an IDE. I think you have may have gotten your variables mixed up here, `player` should probably be replaced with `bPlayer`.